common/defs/ethereum/networks.json

```diff
@@ -367,5 +367,25 @@
     "rskip60": false,
     "url": "https://pirl.io",
     "blockbook": []
+  },
+{
+    "chain": "volta",
+    "chain_id": 73799,
+    "slip44": 44,
```

Comment:
volta is testnet, correct? please use `slip44` id `1`

Reply:
ok - I'll change that...